model/__init__.py

```diff
@@ -298,6 +298,14 @@ def secrets_server(self, cfg_name=None):
     def protecode(self, cfg_name=None):
         return self.cfg_factory.protecode(self._default_name('protecode', cfg_name))
 
+    def iter_configs(self, cfg_type_name):
```

Comment:
add a docstring stating the contract:
-> returns a generator yielding all config_elements for the given cfg_type_name
add type-hint for cfg_type_name
state that ValueError be risen in case the cfg_type_name does not exist

Reply:
this function should be mv'ed to the cfg_factory. it may be re-used from cfg_factory here.